src/input-range-select/input-range-select.tsx

```diff
@@ -291,6 +299,15 @@ export const InputRangeSelect = <T, V>({
                     {renderSelectorContent("from")}
                     {renderSelectorContent("to")}
                 </RangeInputInnerContainer>
+                <ClearContainer onClick={handleClear} type="button">
```

Comment:
1. should check for readOnly and disabled states as well
2. the conditional rendering logic should include the ClearContainer, otherwise there is still an invisible button that can clicked/tabbed to
3. does this component always need to show the clear button? if not, you can introduce a prop called `allowClear`

Reply:
3. yes this component always need to show the clear button.